### ISSUE TITLE
Android viruses not getting scanned

### ISSUE THREAD

**Issue by ghost** [VISWESWARAN1998/CyberGod-KSGMPRH#2]
Dear the whole code is fine but it is not detecting the virus files from android which are identified by avast and other antiviruses. 
How to add up the extensions of files, virus definitions and signatures in order to prevent android from malicious files.

**Comment by VISWESWARAN1998** [comment]
@ALiz22 
Yes, this is because the database has less signatures. This project is almost dead due to lack of support.

**Comment by VISWESWARAN1998** [comment]
New desktop application "2018 rework" is more mature however.

**Comment by duchoak15** [comment]
> New desktop application "2018 rework" is more mature however.

Why does not New desktop application "2018 rework" update database?

**Comment by duchoak15** [comment]
> > New desktop application "2018 rework" is more mature however.
> 
> Why does not New desktop application "2018 rework" update database?

can you help me update database section? please

**Comment by kuchayyasir** [comment]
@VISWESWARAN1998 I m not able to run this project can u help me out

**Comment by afiqbilal** [comment]
> > > New desktop application "2018 rework" is more mature however.
> > 
> > 
> > Why does not New desktop application "2018 rework" update database?
> 
> can you help me update database section? please

are you able to run custom scan folder and dupe clean in 2018 rework??

**Comment by VISWESWARAN1998** [comment]
@duchoak15 It is yet to be done. Will soon add this into the project.

**Comment by VISWESWARAN1998** [comment]
The wait is over new project will solve all your problems.

https://github.com/VISWESWARAN1998/Mrida

**Comment by VISWESWARAN1998** [comment]
@ALiz22 @duchoak15 See the new repo and download the latest release: https://github.com/VISWESWARAN1998/Mrida

**Comment by VISWESWARAN1998** [comment]
> 
> 
> @VISWESWARAN1998 I m not able to run this project can u help me out

Please switch over to new project
